.chloggen/jgm-issue-22850.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
```

Comment:
Since this is now just adding a test we can remove the changelog entry and just add `[chore]` to the beginning of the PR title.

Reply:
Done. I've also rebased and squashed.